### PR TITLE
[HOLD] RHD-644 - Red Hat Mobile Overview page

### PR DESCRIPTION
**Pull request by Dantheman720**
Work in Progress

### CODE REVIEW

**Comment by Dantheman720** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

File: _brms-quickstarts [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 66df6d3774cf1c1e978a23ef0640d3e65d9688e0
+Subproject commit 7fb9b53309204cd3d2040306a5bfdbd95706ac63
```
Comment (paulrobinson) [inline _brms-quickstarts]: Why are you updating this?

File: _layouts/product-overview.html.slim [hunk 2]
```diff
@@ -31,9 +31,12 @@ h2#overview Overview
 
   .large-11.columns.product-video
     = video_player(video(page.product.intro_video)) if page.product.intro_video
+    - if page.product.intro_youtube
+      = video_player(site.videos["#{page.product.intro_youtube}"])
```
Comment (paulrobinson) [inline _layouts/product-overview.html.slim]: Can you match the above syntax? It's hard to read with the mixture. E.g:

        = video_player(site.videos["#{page.product.intro_youtube}"]) if page.product.intro_youtube

Also, would this work?

       = video_player(video(page.product.intro_youtube)) if page.product.intro_youtube

File: products/mobileplatform/_common/featured_webinar.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+:awestruct-interpolate: true
```
Comment (paulrobinson) [inline products/mobileplatform/_common/featured_webinar.adoc]: I don't think this file is needed

File: _layouts/product-overview.html.slim [hunk 4]
```diff
@@ -50,6 +53,11 @@ br
     div(class="feature-text")
       = page._features3
 
+- if page._special_table
```
Comment (paulrobinson) [inline _layouts/product-overview.html.slim]: Why can't you put the table in the 'detail' section, like we do for Fuse: https://github.com/redhat-developer/developers.redhat.com/blob/master/products/fuse/overview.adoc#detail

Comment (Dantheman720) [inline _layouts/product-overview.html.slim]: Good point.  Originally, it caused problems with the layout.  I will test with detail.

File: _layouts/product-overview.html.slim (hunk 2 above)
Comment (Dantheman720) [inline _layouts/product-overview.html.slim]: I tried that and kept getting an issue with the 'duration' metadata.

File: _brms-quickstarts (hunk 1 above)
Comment (Dantheman720) [inline _brms-quickstarts]: I can back that out.  It was updated when I ran 'rake update'

Comment (Dantheman720) [inline _brms-quickstarts]: I am not able to remove this for some reason.

**Comment by paulrobinson** [comment]
Thanks Dan, I've now taken this over to get the finishing touches done before it goes live today.

See: https://github.com/redhat-developer/developers.redhat.com/pull/402

**Pull request closed without merging**